.travis.yml

```diff
@@ -1,7 +1,3 @@
-####################################################################################
```

Comment:
Yes, @StephenWeatherford, this is intentional for our fork, as that enables us to do testing of each example via the `deploy.ci.sh`. We will be reverting those changes back for our PRs into hashicorp/terraform.